### PR TITLE
update impl pseudocode in chapter voting

### PR DESCRIPTION
**Pull request by walkeronwater**
minor changes + impl pseudocode

### CODE REVIEW

File: 08_voting/index.tex [hunk 1]
```diff
@@ -105,6 +107,113 @@ \section{Voting Protocol}
...
+\label{sec:VotingImplementation}
+In IIGO sessions, the server acts as an intermediary to receive the parameters:
+\begin{itemize}
+    \item the purposes\footnote{When the voting for changes of rules is called, the name of rule to be voted on is passed to the server.} of voting.
+    \item the identities of islands.
+    \item the ballots from islands.
+    \item the voting method that is chosen.
+\end{itemize}
+
+When receives all the ballots, the server counts the votes and selects a winner according to the voting-method parameter (see Algorithm~\ref{algo:Plurality}, \ref{algo:Borda}, \ref{algo:Runoff}, \ref{algo:InstantRunoff} and \ref{algo:Approval}). The recommended\footnote{The declaration of winner is subject to the islands' strategies where the initiator of voting may or may not abuse the power to declare a winner without following the voting method that is chosen.} winner is available for the initiator of voting.
```
Comment (NGovani) [inline 08_voting/index.tex]:
```suggestion
Once it receives all the ballots, the server counts the votes and selects a winner according to the voting-method parameter (see Algorithm~\ref{algo:Plurality}, \ref{algo:Borda}, \ref{algo:Runoff}, \ref{algo:InstantRunoff} and \ref{algo:Approval}). The recommended\footnote{The declaration of winner is subject to the islands' strategies where the initiator of voting may or may not abuse the power to declare a winner without following the voting method that is chosen.} winner is available for the initiator of voting.
```

**Review by NGovani — COMMENTED** [review]
Psuedocode looks goods minor changes

File: 08_voting/index.tex (hunk 1 above)
Comment (walkeronwater) [reply 08_voting/index.tex]: changed

**Review by NGovani — APPROVED** [review]
LGTM

File: 08_voting/index.tex [hunk 2]
```diff
@@ -105,6 +107,113 @@ \section{Voting Protocol}
...
+    \item the purposes\footnote{When the voting for changes of rules is called, the name of rule to be voted on is passed to the server.} of voting.
+    \item the identities of islands.
+    \item the ballots from islands.
+    \item the voting method that is chosen.
+\end{itemize}
+
+Once it receives all the ballots, the server counts the votes and selects a winner according to the voting-method parameter (see Algorithm~\ref{algo:Plurality}, \ref{algo:Borda}, \ref{algo:Runoff}, \ref{algo:InstantRunoff} and \ref{algo:Approval}). The recommended\footnote{The declaration of winner is subject to the islands' strategies where the initiator of voting may or may not abuse the power to declare a winner without following the voting method that is chosen.} winner is available for the initiator of voting.
+
+\begin{algorithm}[!h]
+\KwData{IslandID, ballots("Aye", "Nay" or "Abstain")}
```
Comment (lhl2617) [inline 08_voting/index.tex:125]: \`\`quotes''

Comment (lhl2617) [reply 08_voting/index.tex:125]: "quotes" <- wrong direction

Comment (walkeronwater) [reply 08_voting/index.tex:125]: changed in #47

**Pull request merged**